LayoutTests/platform/mac/TestExpectations

```diff
@@ -979,6 +979,7 @@ webkit.org/b/181832 inspector/unit-tests/throttler.html [ Pass Failure ]
 webkit.org/b/161361 inspector/unit-tests/yieldable-task.html [ Pass Failure ]
 webkit.org/b/169228 inspector/worker/console-basic.html [ Pass Timeout ]
 webkit.org/b/164872 inspector/worker/debugger-multiple-targets-pause.html [ Pass Failure Timeout ]
+webkit.org/b/164872 inspector/worker/debugger-pause-subworker.html [ Pass Failure Timeout ]
```

Comment:
Why do we expect the test to Fail?

Reply:
i think it's for a similar reason as the linked bug, but i cant really remember for sure.  i can try testing it again locally